Project.toml

```diff
@@ -0,0 +1,32 @@
+name = "GeneticVariation"
+uuid = "9bc6ac9d-e6b2-5f70-b0a8-242a01662520"
+authors = ["Ben J. Ward <benjward@protonmail.com>"]
+version = "0.4.1"
+
+[deps]
+Automa = "67c07d97-cdcb-5c2c-af73-a7f9c32a568b"
+BGZFStreams = "28d598bf-9b8f-59f1-b38c-5a06b4a0f5e6"
+BioCore = "37cfa864-2cd6-5c12-ad9e-b6597d696c81"
+BioSequences = "7e6ae17a-c86d-528c-b3b9-7f778a29fe59"
+BufferedStreams = "e1450e63-4bb3-523b-b2a4-4ffa8c0fd77d"
+Combinatorics = "861a8166-3701-5b0c-9a16-15d98fcdc6aa"
+IntervalTrees = "524e6230-43b7-53ae-be76-1e9e4d08d11b"
+Twiddle = "7200193e-83a8-5a55-b20d-5d36d44a0795"
+
+[compat]
```

Comment:
How sure about these compat bounds are you? I suppose we can just set the bounds to current versions, and then test and relax things if anyone complains.

Reply:
I've checked and relaxed the compat bounds.
The biggest issue is incompatibility with BioSequences v2. But that requires larger changes and I think that's better to solve in another release. (And a better approach might be to split out a VCF.jl from this package that will simplify the compatibility situation.)

Reply:
Yeah, fixing that seems outside the scope of this PR.